### PR TITLE
Update of Apache (+PHP) chapter (bsc#1177654)

### PR DESCRIPTION
**Pull request by tbazant**
### PR creator: Description

Some info in the Apache chapter is outdated, this update tries to align it.
!!!IMPORTANT: check the &php-version; entity after backporting

### PR creator: Are there any relevant issues/feature requests?
Fixes https://bugzilla.suse.com/show_bug.cgi?id=1177654

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [x] SLE 15 SP4/openSUSE Leap 15.4 *(current `main`, no backport necessary)*
  - [x] SLE 15 SP3/openSUSE Leap 15.3
  - [x] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
  - [ ] SLE 15 GA
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4
  - [ ] SLE 12 SP3

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

File: xml/apache2.xml [hunk 1]
```diff
@@ -366,11 +375,19 @@
...
+        Configuration directives for serving pages over HTTP2 connection.
+       </para>
+      </listitem>
+     </varlistentry>
      <varlistentry>
       <term><filename>server-tuning.conf</filename></term>
       <listitem>
        <para>
-        Contains configuration directives for the different MPMs (see
+        Contains configuration directives for the different M.PMs (see
```
Comment (tbazant) [inline xml/apache2.xml:390]:
```suggestion
        Contains configuration directives for the different MPMs (see
```

File: xml/apache2.xml [hunk 2]
```diff
@@ -9,13 +9,16 @@
...
-    (<link xlink:href="http://www.apache.org/"/>), it is available for most
-    operating systems. &productnamereg; includes Apache version 2.4. In this
-    chapter, learn how to install, configure and set up a Web server; how to
-    use SSL, CGI, and additional modules; and how to troubleshoot Apache.
+    According to the surveys from <link xlink:href="http://www.netcraft.com/"/>
+    and <link xlink:href="https://w3techs.com/"/>, the Apache HTTP Server
+    (Apache) is one of the world's most popular Web servers. Developed by the
+    Apache Software Foundation
+    (<link
+     xlink:href="http://www.apache.org/"/>), it is available for
```
Comment (dariavladykina) [inline xml/apache2.xml:17]:
```suggestion
    (<link xlink:href="http://www.apache.org/"/>), it is available for
```

File: xml/apache2.xml [hunk 3]
```diff
@@ -109,14 +112,8 @@
...
-<screen>&prompt.sudo;systemctl enable apache2</screen>
-   <para>
-    <remark>taroth 2014-02-11: commenting the following procedure as it is unsure
-      how to enable a service for certain targets with &yast; Services Manager,
-      filed bnc# 863333 for it
-     UPDATE tbazant 2015-08-20: still not clear, leaving commented out
-     </remark>
-   </para>
+<screen>&prompt.sudo;systemctl enable apache2.service</screen>
+   <para></para>
```
Comment (dariavladykina) [inline xml/apache2.xml:116]:
```Remove this?
   <para></para>
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please see some suggestions

File: xml/apache2.xml (hunk 3 above)
[the remark this answers is not in the document]
Comment (dariavladykina) [reply xml/apache2.xml:116]: Remove  <para></para>?

**Pull request merged**